src/nvim/tui/tui.c

```diff
@@ -1642,6 +1640,15 @@ static void invalidate(TUIData *tui, int top, int bot, int left, int right)
   }
 }
 
+static void ensure_space_buf_size(TUIData *tui, int len)
+{
+  if (len > tui->space_buf_len) {
+    tui->space_buf = xrealloc(tui->space_buf, (size_t)len * sizeof *tui->space_buf);
+    memset(tui->space_buf, ' ', (size_t)len);
```

Comment:
The `memset()` doesn't use `sizeof`, so I think the `xrealloc()` shouldn't use `sizeof` either.
```suggestion
    tui->space_buf = xrealloc(tui->space_buf, len);
    memset(tui->space_buf, ' ', len);
```